Model/Transaction.php

```diff
@@ -276,7 +276,10 @@ protected function buildLineItems($order, $items, $type = 'order') {
 
             $product = $this->productRepository->getById($item->getProductId(), false, $order->getStoreId());
 
-            if ($product->getTaxClassId()) {
+            // Check for a PTC assigned directly to the product; otherwise fall back to tax classes
+            if ($product->getTjPtc()) {
+                $lineItem['product_tax_code'] = $product->getTjPtc();
+            } elseif ($product->getTaxClassId()) {
```

Comment:
Just to confirm, the $product here would be the child in the case of a configurable product? So no need to pull the child products PTC here?

Reply:
@dallendalton Good catch.  I'd mistakenly assumed that we skip the configurable products and this would be a simple product, but the opposite is true.  Fix in the most recent commit.